app/views/checkout/_terms_and_conditions.html.haml

```diff
@@ -1,2 +1,5 @@
-%p.small
-  = t('.message_html', terms_and_conditions_link: link_to( t( '.link_text' ), current_order.distributor.terms_and_conditions.url, target: '_blank')) if current_order.distributor.terms_and_conditions.file?
+- terms_and_conditions_activated = current_order.distributor.terms_and_conditions.file?
```

Comment:
IMO this would be better off as a view helper so we don't couple the partial with this logic. It's also testable that way.

Reply:
Looks like you did it already in https://github.com/openfoodfoundation/openfoodnetwork/pull/6000/ ?

Reply:
correct, this was done in 6000 and this is the partial after it: https://github.com/openfoodfoundation/openfoodnetwork/pull/6000/files#diff-cd5c1c15d07aa37fb1e5d15f4ffe60fe6b4ee1084d6f7e5be2c313925a901d06

Reply:
ping @sauloperez can you re-review this please?